crates/batch-prover/src/proving.rs

```diff
@@ -160,8 +160,8 @@ where
             })?
             .expect("There should be a state root");
 
-        let state_transition_data: StateTransitionData<StateRoot, Witness, Da::Spec> =
-            StateTransitionData {
+        let state_transition_data: BatchProofCircuitInput<StateRoot, Witness, Da::Spec> =
```

Comment:
rename the variables you can catch as well.

Reply:
👍